compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEOps.td

```diff
@@ -1749,15 +1750,18 @@ def AMDAIE_DMAStartOp: AMDAIE_Op<"dma_start", [
 
   let summary = "An op to start DMA";
   let arguments = (
-    ins Index:$tile,
-        DMAChannelDir:$channel_dir,
-        ConfinedAttr<I8Attr, [IntMinValue<0>]>:$channel_index,
+    ins Index:$channel,
```

Comment:
Maybe add some documentation to the op description about the need for the connection op.